core/vm/analysis.go

```diff
@@ -79,6 +79,13 @@ func codeBitmapInternal(code, bits bitvec) bitvec {
 		if int8(op) < int8(PUSH1) { // If not PUSH (the int8(op) > int(PUSH32) is always false).
 			continue
 		}
+		// Short circruit for now on EOF ops with immediates.
+		// TODO(matt): make EOF-specific code bitmap
+		if op == RJUMP || op == RJUMPI || op == CALLF {
+			bits.setN(set4BitsMask, pc)
+			pc += 4
+			continue
+		}
```

Comment:
What's this with `4` bytes? The immedate of `RJUMP`/RJUMPI` (https://eips.ethereum.org/EIPS/eip-4200) is `16` bits (2 bytes).  

And so does `CALLF`: https://eips.ethereum.org/EIPS/eip-4750 

> Has one immediate argument,code_section_index, encoded as a 16-bit unsigned big-endian value.


```suggestion
		if op == RJUMP || op == RJUMPI || op == CALLF {
			bits.setN(set2BitsMask, pc)
			pc += 2
			continue
		}
```

( ignoring the fact that right now, this would be a consensus-error on legacy code, and needs to be in it's own `analyzeEofCode` )